build.gradle

```diff
@@ -251,7 +252,13 @@ subprojects {
       // since the latter is excluded (see the comment to `jgit` for more nuances).
       // The below dependency provides both slf4j-api and an implementation, both already in the same version (1.7.30 i.e. the current stable).
       dependencies {
-        testImplementation group: 'org.slf4j', name: 'slf4j-simple', version: slf4jVersion
+        testImplementation group: 'org.slf4j', name: 'slf4j-simple', version: slf4jTestImplVersion
```

Comment:
To be super-correct/safe, I'd actually go back to having a single `slf4jVersion` for both `Api` and `TestImpl` - sorry for advicing otherwise before, I now see it's actually detrimental to classpath sanity.